docs/module_types/common_generate_module_properties.md

```diff
@@ -65,6 +65,12 @@ When `true` one job will run at a time - they won't be concurrent.
 Additional include paths to add for modules that use `generated_headers`. This
 will be defined relative to the module-specific build directory.
 
+----
+### **bob_generated.encapsulates** (optional)
+A list of source modules that this bob_generated_source will encapsulate.
+When this module is used with generated_headers, the named modules export_gen_include_dirs will be forwarded.
```

Comment:
Grammar: "the named modules' `export_gen_include_dirs`..." (missing apostrophe)

Reply:
Sure, but you probably mean "module's"

Reply:
Nope - there can be multiple modules, so the apostrophe goes after the 's' :)